src/optimagic/optimizers/nevergrad_optimizers.py

```diff
@@ -0,0 +1,139 @@
+from dataclasses import dataclass
+from typing import Literal, Tuple
+
+import numpy as np
+from numpy.typing import NDArray
+
+try:
+    import nevergrad as ng
+except ImportError:
+    pass
+
+from optimagic import mark
+from optimagic.config import IS_NEVERGRAD_INSTALLED
+from optimagic.exceptions import NotInstalledError
+from optimagic.optimization.algo_options import (
+    STOPPING_MAXFUN_GLOBAL,
+)
+from optimagic.optimization.algorithm import Algorithm, InternalOptimizeResult
+from optimagic.optimization.internal_optimization_problem import (
+    InternalOptimizationProblem,
+)
+from optimagic.typing import AggregationLevel, PositiveInt
+
+
+@mark.minimizer(
+    name="nevergrad_oneplusone",
+    solver_type=AggregationLevel.SCALAR,
+    is_available=IS_NEVERGRAD_INSTALLED,
+    is_global=False,
+    needs_jac=False,
+    needs_hess=False,
+    supports_parallelism=True,
+    supports_bounds=True,
+    supports_linear_constraints=False,
+    supports_nonlinear_constraints=True,  # TBA
+    disable_history=False,
+)
+@dataclass(frozen=True)
+class NevergradOnePlusOne(Algorithm):
+    noise_handling: str | Tuple[str, float] | None = None
+    mutation: Literal[
+        "gaussian",
+        "cauchy",
+        "discrete",
+        "fastga",
+        "rls",
+        "doublefastga",
+        "adaptive",
+        "coordinatewise_adaptive",
+        "portfolio",
+        "discreteBSO",
+        "lengler",
+        "lengler2",
+        "lengler3",
+        "lenglerhalf",
+        "lenglerfourth",
+        "doerr",
+        "lognormal",
+        "xlognormal",
+        "xsmalllognormal",
+        "tinylognormal",
+        "lognormal",
+        "smalllognormal",
+        "biglognormal",
+        "hugelognormal",
+    ] = "gaussian"
+    annealing: Literal[
+        "none", "Exp0.9", "Exp0.99", "Exp0.9Auto", "Lin100.0", "Lin1.0", "LinAuto"
+    ] = "none"
+    sparse: bool | int = False
+    super_radii: bool = False
+    smoother: bool = False
+    roulette_size: int = 2
+    antismooth: int = 55
+    crossover: bool = False
+    crossover_type: Literal["none", "rand", "max", "min", "onepoint", "twopoint"] = (
+        "none"
+    )
+    tabu_length: int = 0
+    rotation: bool = False
+
+    stopping_maxfun: PositiveInt = STOPPING_MAXFUN_GLOBAL
+    n_cores: PositiveInt = 1
+
+    def _solve_internal_problem(
+        self, problem: InternalOptimizationProblem, x0: NDArray[np.float64]
+    ) -> InternalOptimizeResult:
+        if not IS_NEVERGRAD_INSTALLED:
+            raise NotInstalledError(
+                "Nevergrad is not installed. Please install it to use this algorithm."
+            )
+        param = ng.p.Array(
+            init=np.clip(x0, problem.bounds.lower, problem.bounds.upper)
```

Comment:
Why did you decide to clip the start parameters? Optimagic ensures that x0 is within bounds. Did you have a problem when not doing the clipping?